image/image.go

```diff
@@ -120,7 +120,7 @@ func unpack(w walker, dest, refName string) error {
 		return err
 	}
 
-	if err = ref.validate(w, validRefMediaTypes); err != nil {
+	if err = ref.validate(w, []string{v1.MediaTypeImageManifest}); err != nil {
```

Comment:
`manifest list` points to only a few special `manifests`, but specify its `platform`, there is no other property requirements. I do not think this is what you want to get, and in the instructions, `ref` can only point to the `manifest`.